apps/openmw/mwgui/trainingwindow.cpp

```diff
@@ -168,23 +168,25 @@ namespace MWGui
         MWMechanics::NpcStats& pcStats = player.getClass().getNpcStats(player);
 
         const MWWorld::ESMStore& store = *MWBase::Environment::get().getESMStore();
+        float playerSkill = getPlayerSkillForTraining(pcStats, skill->mId);
 
-        int price = pcStats.getSkill(skill->mId).getBase()
+        int price = playerSkill
             * store.get<ESM::GameSetting>().find("iTrainingMod")->mValue.getInteger();
         price = MWBase::Environment::get().getMechanicsManager()->getBarterOffer(mPtr, price, true);
 
+        price = std::max(1, price);
```

Comment:
If the player drains their skill by 100, the price here will get set to 0. 

However, the code that generates the training list sets the minimum price to 1gp. This creates a situation where the price says 1gp, but actually costs the player 0. 

I set the minimum here to 1 for consistency, but I'm open to handling this differently, if anyone has thoughts/concerns.